Dockerfile

```diff
@@ -1,13 +1,13 @@
 #FROM databoxsystems/base-image-ocaml:alpine-3.4_ocaml-4.04.2 as BUILDER
-FROM ocaml/opam:alpine-3.6_ocaml-4.04.2 as BUILDER
+FROM ocaml/opam-dev:alpine-3.6_ocaml-4.04.2 as BUILDER
```

Comment:
`ocaml/opam-dev` or `ocaml/opam2` now?

Reply:
Yep `opam2` works, though alpine needs to be bumped to `alpine-3.7`.

Optimally it would be using `alpine-3.10` but for some reason `3.10` fails to be able to resolve the dependencies while `3.7` does...